### PR TITLE
Feature/leak resolution manual segregation

### PR DESCRIPTION
**Pull request by dadamsretrorabbit**
Removes Water Sensor to Sensor.

### CODE REVIEW

File: WaterLog_Backend/Controllers/SegmentLeaksController.cs [hunk 1]
```diff
@@ -102,15 +102,24 @@ public async Task<ActionResult<SegmentLeaksEntry>> Resolve([FromForm] int id)
             }
             else if(leaks.ResolvedStatus == EnumResolveStatus.UNRESOLVED)
             {
-                leaks.ResolvedStatus = EnumResolveStatus.RESOLVED;
+                var latestEntry = await _db.HistoryLogs.Where(a => a.EventsId == leaks.SegmentsId && a.Type == EnumTypeOfEvent.LEAK).OrderByDescending(b => b.CreationDate).FirstOrDefaultAsync();
```
Comment (FredJEhlers) [inline WaterLog_Backend/Controllers/SegmentLeaksController.cs]: Split over multiple lines

File: WaterLog_Backend/Controllers/SegmentEventsController.cs [hunk 2]
```diff
@@ -154,7 +154,7 @@ public async Task<List<GetAlerts>> GetAlertsByPage(int id = 1)
                                 .FaultDate.
                                 Subtract(entry
                                 .AttendedDate)),
-                                ((entry.SensorType == EnumSensorType.WATER_FLOW_SENSOR) ? "Water Sensor" : "Sensor"),
+                                "Sensor",
```
Comment (FredJEhlers) [inline WaterLog_Backend/Controllers/SegmentEventsController.cs]: You are using this a lot along with faulty, consider constants

**Review by FredJEhlers — APPROVED** [review]

File: WaterLog_Backend/Controllers/SegmentEventsController.cs (hunk 2 above)
Comment (dadamsretrorabbit) [reply WaterLog_Backend/Controllers/SegmentEventsController.cs]: Fixed

File: WaterLog_Backend/Controllers/SegmentLeaksController.cs (hunk 1 above)
Comment (dadamsretrorabbit) [reply WaterLog_Backend/Controllers/SegmentLeaksController.cs]: Done

**Pull request merged**